### PR TITLE
Example .NET POST request

### PR DESCRIPTION
**Pull request by tmct**
This adds a simple .NET example of a POST to the Yudu API.

@jrdh @THJSmithSW

### CODE REVIEW

File: examples/dotNetClient/DotNetYuduAPIExample/Program.cs [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+using System.Security.Cryptography;
+using System.Text;
+using RestSharp;
+
+namespace DotNetYuduAPIExample
+{
+    internal class Program
+    {
+        private const string SecretKey = "Please contact Yudu to obtain this shared secret.";
+        private const string ApiKey = "Please contact Yudu to obtain this API key.";
```
Comment (jrdh) [inline examples/dotNetClient/DotNetYuduAPIExample/Program.cs]: Clients don't necessarily need to contact Yudu for this - if they know where to find it when they login to Publisher or have already been given it by Yudu they can just enter the details.

**Comment by jrdh** [comment]
Looks good to me! :+1:

File: examples/dotNetClient/DotNetYuduAPIExample/Program.cs (hunk 1 above)
Comment (tmct) [inline examples/dotNetClient/DotNetYuduAPIExample/Program.cs]: Yeah, was just copying https://github.com/yudugit/rest-api-documentation/blob/master/examples/phpClient/examples.php :p

How about "Please contact Yudu if you don't already have this..."?

**Comment by rmanthorpe** [comment]
Good call. Mist of your clients seem to use .NET :+1:

File: examples/dotNetClient/DotNetYuduAPIExample/Program.cs (hunk 1 above)
Comment (jrdh) [inline examples/dotNetClient/DotNetYuduAPIExample/Program.cs]: Sure :+1:

**Pull request merged**